crawler.js

```diff
@@ -405,14 +409,30 @@ class Crawler {
   }
 
   async extractLinks(page, seedId, depth, selector = "a[href]") {
-    let results = [];
+    const results = [];
```

Comment:
should this be a const if we're pushing results to it? would let me more correct?

Reply:
the way JS works, adding to the array is still ok for const, it just won't allow re-assigning to the results objects.